botcity/maestro/sdk.py

```diff
@@ -397,6 +397,8 @@ def create_task(self, activity_label: str, parameters: Dict[str, object],
             test: Whether or not the task is a test.
             priority: (Optional[int], optional) An integer from 0 to 10 to refer to execution priority.
             min_execution_date (Optional[datetime.datetime], optional): Minimum execution date for the task.
+                The BotCity Orchestrator time zone is in UTC-0. Therefore, consider the difference between
+                time zones when using this parameter.
 
```

Comment:
What do you think about working with a warning? It would be more evident to the user when reading the documentation and even in the IDE.

```
        Warning:
                The BotCity Orchestrator time zone is in UTC-0. Therefore, consider the difference between
                time zones when using this parameter.
        
```

![image](https://github.com/botcity-dev/botcity-maestro-sdk-python/assets/59173212/11050fc9-81cc-40b1-9f35-10b0185a1c27)

(Example of docstring in the web framework in the stop_browser function)